src/test/vvv/InventoryTest.java

```diff
@@ -0,0 +1,195 @@
+package vvv;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class InventoryTest {
+    final int ITEM_COUNT_PER_SLOT = 2;
+    final int EQUIPMENT_COUNT_PER_SLOT = 3;
+    Inventory inventory;
+
+    @BeforeEach
+    void init() {
+        inventory = new Inventory();
+        inventory.addSlot(new AminoAcidSlot(ITEM_COUNT_PER_SLOT));
+        inventory.addSlot(new EquipmentSlot());
+    }
+
+    @Test
+    void fillAll_OneSlot_Succeed() {
+        // Arrange
+        Slot s = getFilledSlot();
+
+        // Act
+        inventory.fillAll(s);
+
+        // Assert
+        assertEquals(ITEM_COUNT_PER_SLOT, inventory.getAminoAcidCount());
```

Comment:
Please write message to assert so if it fails it tells what failed

Reply:
The environment tells where the failure happened.